### PR TITLE
Feedback fixes

### LINKED ISSUE #85: Feedback : Update button point to an empty form
We should fill the form with previous rating.

### LINKED ISSUE #122: Should redirect user to the repository page from the feedback page when he is a contributor
A contributor should not have access to the feedback form.

### PR DESCRIPTION
**Pull request by brunnoattorre**
Fix #122 Fix #85

### CODE REVIEW

File: app/controllers/ApplicationController.scala [hunk 1]
```diff
@@ -110,10 +113,20 @@ class ApplicationController @Inject()(
     val repoName: String = owner + "/" + repositoryName
     repoService.getFromNeoOrGitHub(request.identity, repoName).flatMap({
       case Some(repository) =>
-        repoService.canUpdateFeedback(repoName, request.identity).map(canUpdate =>
-          Ok(views.html.feedbackForm(gitHubProvider, request.identity)(owner, repositoryName, FeedbackForm.form, canUpdate))
-        )
-      case None => Future(NotFound(views.html.error("notFound", 404, "Not Found",
+        request.identity match {
+          case Some(user) => owner match {
+            case user.username => Future(Redirect(routes.ApplicationController.gitHubRepository(owner, repositoryName, None).url))
```
Comment (callicles) [inline app/controllers/ApplicationController.scala]: We want the redirection if the user has contributed to the repo, not only when his name matches the owner name.

Comment (brunnoattorre) [inline app/controllers/ApplicationController.scala]: Hey @callicles , I remember this functionality was in place before but we've changed it for some reason (Check https://github.com/gitlinks/gitrank-web/commit/3113558a532dc9fd37c219b6011b427a1843dcf3#diff-98d8d38b24fc27a681a4b2357fd63e87 ). We used to check if the user had some contributions (check canAddFeedback) and redirect to repo page.  
That's why I thought we are going to use this only for the owners of the repo. I've made the fix for redirecting but want your confirmation before pushing the new version

Comment (callicles) [inline app/controllers/ApplicationController.scala]: Hum you are right. I don't remember why we changed. I feel like there was something not really working, but not sure. 

Let's try to put it as required by the feedback we got. (Redirect for every contributor)
Thank you

Comment (brunnoattorre) [inline app/controllers/ApplicationController.scala]: @callicles Ok, pushed corrections

File: app/controllers/ApplicationController.scala [hunk 2]
```diff
@@ -37,6 +38,8 @@ class ApplicationController @Inject()(
                                        quickstartService: QuickstartService)
   extends Silhouette[User, SessionAuthenticator] {
 
+
+  val STATUS_NOT_FOUND = 404
```
Comment (callicles) [inline app/controllers/ApplicationController.scala]: Codacy Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/gitlinks/gitrank-web/file/1393106637/issues/source?bid=2432310&fileBranchId=2979982#l42)

File: app/controllers/ApplicationController.scala [hunk 3]
```diff
@@ -37,6 +38,8 @@ class ApplicationController @Inject()(
                                        quickstartService: QuickstartService)
   extends Silhouette[User, SessionAuthenticator] {
 
+
+  val statusNotFound = 404
```
Comment (callicles) [inline app/controllers/ApplicationController.scala]: Isn't there already a constant defined for 404 ?

**Pull request merged**